lib/database_cleaner/safeguard.rb

```diff
@@ -42,7 +44,8 @@ class RemoteDatabaseUrl
       LOCAL = %w(localhost 127.0.0.1)
 
       def run
-        raise Error::RemoteDatabaseUrl if !skip? && given?
+        return if skip?
+        raise Error::RemoteDatabaseUrl if given?
```

Comment:
I split this across two lines to match the other check classes in this file. It also helped me understand the second line a lot more easily.